src/main/kotlin/com/chloe/kotlinserv/route/PostGeoDataRoute.kt

```diff
@@ -32,14 +32,14 @@ class PostGeoDataRoute @Inject constructor(
 
                 geoDataServiceImpl.addToList(data, requestHeaders?.firstOrNull())
 
-                HttpResponse(
+                return HttpResponse(
                     code = 200,
                     responseBody = null,
                     contentType = mapOf("content-type" to "application/json")
                 )
             } catch (e: JsonSyntaxException) {
```

Comment:
Add one more catch for Throwable to handle all other errors. Respond with 500 code